lib/ex_doc/formatter/html.ex

```diff
@@ -390,6 +384,24 @@ defmodule ExDoc.Formatter.HTML do
     |> Enum.sort_by(fn extra -> GroupMatcher.group_index(groups, extra.group) end)
   end
 
+  def generate_redirects(config, ext) do
+    config.redirects
+    |> Map.put_new("index", config.main)
```

Comment:
This will fail if we give it a list, so we need to add a test, replace `Enum.map` by `Map.new`, and call it at the end.

Reply:
Ah, sorry about that, fixed  and tested :)